### DISCUSSION TITLE
Libre pro support

### DISCUSSION THREAD

**Discussion by prakashdevadas69** [NightscoutFoundation/xDrip#1930] [Q&A]
Is it possible to have support for libre pro.
We get only libre pro in india

**Comment by Navid200** [comment]
Can you provide a link to the product?

**Reply by prakashdevadas69** [reply]
Hi,
Can this libre support be part of the current xDrip / xDrip4iO5 code.
I did not get your message to provide link for the product, could you pls clarify

**Reply by Navid200** [reply]
The only way a developer can add support for a device is to know about the device.
You have provided a name.  Can you provide anything else?  
Let's say I want to get a sample to give to a developer.  How can I do that if I don't know where to buy one?
Where can one buy one of these?  A developer needs to test one before they can submit a change to the code.

**Reply by prakashdevadas69** [reply]
Thanks for the clarification. I do not know whether Libre Pro is available in your country. As we don’t get this online here india as well.
I really do not know how to make this sensor available to you.
Kindly advice

**Reply by varunmashru** [reply]
@Navid200 It can be made available to you asap. Let me know how that would work

**Reply by Navid200** [reply]
A developer is reaching out to an individual who had programming skills and had already looked at this to get updates.

**Comment by tzachi-dar** [comment]
Initial support for this is in https://github.com/NightscoutFoundation/xDrip/pull/2075.
I would appreciate if someone can try this code.

**Comment by prakashdevadas69** [comment]
Hi,
I tried but was not able connect to Libre Pro and Miao Miao 3
It was stuck with collecting initial data


Regards

**Comment by varunmashru** [comment]
Oh nice. They're trying as well.

On Wed, Apr 13, 2022 at 8:08 PM prakashdevadas69 ***@***.***>
wrote:

> Hi,
> I tried but was not able connect to Libre Pro and Miao Miao 3
> It was stuck with collecting initial data
>
>
> Regards
>
> > On Apr 13, 2022, at 4:37 AM, tzachi-dar ***@***.***> wrote:
> >
> > ﻿
> > Initial support for this is in #2075.
> > I would appreciate if someone can try this code.
> >
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you authored the thread.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NightscoutFoundation/xDrip/discussions/1930#discussioncomment-2560267>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA3PVPWJR3WFSORDWOLJEQLVE3L65ANCNFSM5L2HXL7Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
- *Varun Mashru*
  User Experience Design Lead at Tcules <https://www.tcules.com>
  Mo. +91 8238 659055
  LinkedIn Profile <https://in.linkedin.com/in/varunmashru>
